neoexchange/core/archive_subs.py

```diff
@@ -147,7 +147,7 @@ def check_for_archive_images(request_id=None, obstype='EXPOSE', limit=3000):
     auth_header = {'Authorization': 'Token {}'.format(settings.ARCHIVE_TOKEN)}
 
     base_url = settings.ARCHIVE_FRAMES_URL
-    archive_url = '%s?limit=%d&REQNUM=%s&OBSTYPE=%s' % (base_url, limit, request_id, obstype)
+    archive_url = '%s?limit=%d&REQNUM=%s&OBSTYPE=%s&OBJECT=%s' % (base_url, limit, request_id, obstype, obj.replace(' ', '+'))
```

Comment:
so, my understanding is that this is basically a search term. It seems to work equally well with slashes and spaces and pluses. I imagine an `&` would break this? Mostly this was to avoid breaks in the url. Ultimately this or something like this is required to sort out the different targets from multi-target blocks. In other cases, the `reqnum` will cover everything.

Reply:
can confirm, `&` breaks things

Reply:
I guess it's a question of how often this is going to happen and someone is going to override the object name with `<rock name> & <standard name> `...

Reply:
I'm not even really sure HOW one could do this. This basically pulls the name of the target. I'll put a catch in anyway.

Reply:
somebody has probably made a stellar naming notation that involves `&` and `#`s...